docs/data-sources/node.md

```diff
@@ -50,32 +50,32 @@ data "wallarm_node" "example" {
 
 ## Argument Reference
 
-`filter` - (Required) Filters set in the `key=value` format used to look up for WAF node details. Possible keys:
+`filter` - (**required**) filters set in the `key=value` format used to look up for Wallarm node details. Possible keys:
 
-- `uuid` - (Optional) WAF node UUID.
-- `hostname` - (Optional) WAF node name.
-- `type` - (Optional) WAF node type. Can be: `cloud_node` for cloud WAF nodes, `node` for regular WAF nodes.
-- `enabled` - (Optional) Indicator of the WAF node status. Can be: `true` for enabled WAF nodes and `false` for disabled WAF nodes.
+- `uuid` - (optional) Wallarm node UUID.
+- `hostname` - (optional) Wallarm node name.
+- `type` - (optional) Wallarm node type. Can be: `cloud_node` for cloud Wallarm nodes, `node` for regular Wallarm nodes.
```

Comment:
the same about node types

Reply:
as for the Wallarm node - is it still the cloud_node key for it?

Reply:
Cloud_node is the key for nodes installed in the cloud environments.